_posts/2019-11-15-italian-vs-polish-drivers.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Italian VS Polish Drivers
+date: 2019-11-15 19:49:33.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+author: Valentino Urbano
+---
+
+[I've written about polish drivers in a different article][1], this time I will write about Italian drivers instead.
+
+- In Italy drivers are way less polite than in Poland.
+
+In Poland if you make a mistake lining up or need some space to manouver most people will understand and help you, in Italy they just ignore you and sometimes even try as much as they can not to leave you any space to merge. That unless they're truck drivers, they're usually good everywhere.
+
+- In Italy drivers don't know how to drive on highways
+
+You notice that you've passed the border into Italy by counting the number of people staying in the left lane even though they're not passing anyone and the right lane is completely empty. I haven't seen any other European country where drivers drive like that so consistently.
+
+[1]: {% post_url 2019-08-18-The-Typical-Polish-Driver %}
```

Comment:
That's just two points :P

Reply:
any other idea? :P

Reply:
Torino and people bonking at cars stopped at red light; always coming in front of you without any warning; I forgot the 3rd...

Reply:
Ok so second as we established yesterday is universal... so 3rd was ignoring speed limits and area c and just paying the fines after. Because Italians are so richhhh.